src/ui/public/i18n/index.js

```diff
@@ -18,11 +18,11 @@
  */
 
 import { uiModules } from 'ui/modules';
-import {
-  I18nProvider,
-  i18nFilter,
-  i18nDirective,
-} from '@kbn/i18n/angular';
+import { i18n } from '@kbn/i18n';
+import { I18nProvider, i18nFilter, i18nDirective } from '@kbn/i18n/angular';
+
+// eslint-disable-next-line no-undef
+i18n.init(__WEBPACK_TRANSLATIONS__);
```

Comment:
__note:__ Honestly I'm not sure whether autoloaded module is the right place for this initialization, it's included into commons bundle so I decided to use it. Are these modules just for Angular or ...? Happy to move that into a better place if there is any.